libs/src/services/ethContracts/serviceProviderFactoryClient.js

```diff
@@ -155,7 +155,8 @@ class ServiceProviderFactoryClient extends ContractClient {
       endpoint: info.endpoint,
       spID: serviceId,
       type: serviceType,
-      blocknumber: info.blocknumber
+      blocknumber: info.blocknumber,
```

Comment:
i know this won't be included in the 0.12.2 bump, but this isn't necessary. goal here is to get cnode libs up to date